apps/accelerate/speedster/docs/en/docs/installation.md

```diff
@@ -30,7 +30,59 @@ Great, now you are now ready to accelerate your model 🚀 Please visit the foll
 
 ## (Optional) Selective installation of Speedster requirements
 
-TODO
+By default, the `auto_installer` installs all the dl frameworks and compilers supported by `Speedster`, but some of these may not be relevant to your use case; therefore, in this section we explain how you can customise the installation of these libraries, avoiding those not needed.
+
+For example, if you want to optimize a model with a PyTorch only optimization pipeline, you would not need to install TensorFlow and TensorFlow-specific compilers.
+
+To customize the libraries installation you have two options:
+- [Use the auto-installer (recommended)](#use-the-auto-installer-recommended)
+- [Install the libraries manually](#manual-installation)
+
+### Use the auto-installer (recommended)
+To understand how to install only the required libraries, let's examine the auto-installer API:
+
+    python -m nebullvm.installers.auto_installer --frameworks <frameworks> --extra-backends <backends> --compilers <compilers>
+
+As you can see, three arguments are supported:
+
+- `--frameworks` is used to specify the deep learning frameworks to be installed. The supported frameworks are: `torch`, `tensorflow`, `onnx` and `huggingface`. If you want to optimize a PyTorch model, here you should select `torch`. You can specify multiple frameworks by separating them with a space. For example, `--frameworks pytorch tensorflow` will install PyTorch and TensorFlow. If you want to install all frameworks, you can use `all` as the argument. For example, `--frameworks all` will install all frameworks. Default: `all`.
+- `--extra-backends` is used to specify the extra deep learning backends to be installed. Each framework specified in the previous command can exploit other dl frameworks (we name them backends) to boost the model performance, for example adding `onnx` as extra backend when optimizing a PyTorch model will enable the model conversion to ONNX and the optimization with all the ONNX supported compilers. The supported backends are: `torch`, `tensorflow` and `onnx`. You can specify multiple backends by separating them with a space. For example, `--extra-backends torch tensorflow` will install PyTorch and TensorFlow. If you want to install all backends, you can use `all` as the argument. For example, `--extra-backends all` will install all backends. If you don't want to install extra backends, you can set `--extra-backends none`. Default: `all`.
```

Comment:
I would put a "." after "to boost the model performance". 

What if I run 
```bash
python -m nebullvm.installers.auto_installer --frameworks pytorch --extra-backends onnx tensorflow --compilers all
```
Would it work?

Reply:
I'll add a control that ensures that the provided extra-backends are supported from the selected frameworks

Reply:
Don't forget to add also a note in the documentation explaining that it is possible to select just the extra-backends supported by the chosen framework

Reply:
Yep added ;)